maquetado/scss/tech.scss

```diff
@@ -28,12 +28,21 @@ $tech-border-radius: 5px;
     height: 100%;
     position: absolute;
     right: 10px;
+    transition: linear 0.15s;
```

Comment:
Try to define only the properties you're going to transition, otherwise, the browser will have to do an extra job when layout properties are modified (i.e. when resizing the browser window).